test/e2e/persistent_volumes.go

```diff
@@ -708,19 +850,19 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 			APIVersion: registered.GroupOrDie(api.GroupName).GroupVersion.String(),
 		},
 		ObjectMeta: api.ObjectMeta{
-			GenerateName: "write-pod-",
+			GenerateName: "client-",
 			Namespace:    ns,
 		},
 		Spec: api.PodSpec{
 			Containers: []api.Container{
 				{
 					Name:    "write-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
-					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "touch /mnt/SUCCESS && (id -G | grep -E '\\b777\\b')"},
+					Command: []string{"/bin/sh", "-c"},
```

Comment:
It works either way.  My reasoning was that `/bin/sh -c` worked as the `Command` which reads over the `Arg` string (containing a line or block of bash commands).  Is it recommended that `-c` go into `Args`?